### DISCUSSION TITLE
Has anyone here implemented refreshing the access token using Dio Interceptor?

### DISCUSSION THREAD

**Discussion by realshovanshah** [cfug/dio#1342] [Q&A]
I'm yet to find an elegant way without using retry count or using a different Dio instance all together. Even interceptor locks are now deprecated. There's many open issues discussing this too. #1286 #50

**Comment by kuhnroyal** [comment]
I am using https://pub.dev/packages/fresh_dio with some manual handling. I have not tried the new `QueuedInterceptor` which can probably be used as well.
